frontend/app/api/questions/[id]/route.ts

```diff
@@ -19,6 +21,38 @@ export async function PUT(req: Request, { params }: { params: { id: number } })
       }
     });
 
+    // if in production, push the question to farcaster
+    if (process.env.NODE_ENV === "production") {
```

Comment:
Instead of checking against production, can we add an env var to enable casting to farcaster? It could help if we need to disable it or roll out this code to product but not have it immediately available. Something like `ENABLE_CASTING_TO_FARCASTER`